metis-core/metis-core-service/src/test/java/eu/europeana/metis/core/execution/TestWorkflowExecutor.java

```diff
@@ -170,8 +169,10 @@ void callNonMockedFieldValue_ExceptionWhenExecuteIsCalled() throws Exception {
     workflowExecution.setWorkflowStatus(WorkflowStatus.INQUEUE);
     workflowExecution.setMetisPlugins(abstractMetisPlugins);
 
+    final EcloudBasePluginParameters ecloudBasePluginParameters = new EcloudBasePluginParameters(null, null,
+        workflowExecution.getEcloudDatasetId(), null);
     doThrow(new ExternalTaskException("Some error")).when(oaipmhHarvestPlugin).execute(
-        any(DpsClient.class), isNull(), isNull(), eq(workflowExecution.getEcloudDatasetId()));
+        any(DpsClient.class), any(EcloudBasePluginParameters.class));
```

Comment:
change to ... same(ecloudBasePluginParameters)... ?
That or remove ecloudBasePluginParameters as it is unused.